modules/calcite/src/test/java/org/apache/ignite/internal/processors/query/calcite/exec/rel/AbstractExecutionTest.java

```diff
@@ -85,6 +85,9 @@ public class AbstractExecutionTest extends GridCommonAbstractTest {
     /** Params string. */
     protected static final String PARAMS_STRING = "Task executor = {0}, Execution strategy = {1}";
 
+    /** */
+    protected static final int IN_BUFFER_SIZE = AbstractNode.IN_BUFFER_SIZE;
```

Comment:
Might be put in a dedicated ticket/

Reply:
But it's a minor fix in test. Moreover it's simplifies the tests and also simplifies the test in current ticket.